### PR TITLE
[ROCm] Forward type names from gpu prefix to cuda prefix

### PR DESCRIPTION
**Pull request by jerryyin**
This PR is a follow-up to the original PR #28343. The reviewer requested to break down the original large PR to a series of smaller ones. According to the plan here, this PR is the second one in the whole series. It is dependent on the infrastructure provided from PR #28564

@chsigg @whchung

### CODE REVIEW

File: tensorflow/core/util/gpu_launch_config.h [hunk 1]
```diff
@@ -146,7 +147,7 @@ inline CudaLaunchConfig GetCudaLaunchConfig(int work_element_count,
   return GetGpuLaunchConfig(work_element_count, d);
 }
 
-// Calculate the Cuda launch config we should use for a kernel launch. This
+// Calculate the Gpu launch config we should use for a kernel launch. This
```
Comment (chsigg) [inline tensorflow/core/util/gpu_launch_config.h]: GPU instead of Gpu (admitted, CUDA wasn't cased correctly either).

Comment (jerryyin) [reply tensorflow/core/util/gpu_launch_config.h]: Is the idea to replace every instance of "Gpu" to "GPU" in the comments? I'd assume in the other review #28568, update "Cuda" to "CUDA" in all comments as well. Just double check to make sure.

BTW it make me curious whether this is this part of a rule of a coding convention?

Comment (jerryyin) [reply tensorflow/core/util/gpu_launch_config.h]: Updated all instance of "Gpu" to "GPU" in the comments, let me know if this is expected. Thanks

Comment (chsigg) [reply tensorflow/core/util/gpu_launch_config.h]: Correct, GPU and CUDA in comments, Gpu and Cuda in code. From https://google.github.io/styleguide/cppguide.html:

> When abbreviations or acronyms appear in such names, prefer to capitalize the abbreviations or acronyms as single words (i.e StartRpc(), not StartRPC()).

**Review by chsigg — APPROVED** [review]
Perfect, thanks!

File: tensorflow/core/util/gpu_launch_config.h (hunk 1 above)
Comment (jerryyin) [reply tensorflow/core/util/gpu_launch_config.h]: Thanks for the reply! I'll need get myself a bit more familiar with the cpp style guide.

**Pull request merged**